package/MDAnalysis/coordinates/memory.py

```diff
@@ -302,11 +303,15 @@ def __init__(self, coordinate_array, order='fac',
         .. versionchanged:: 0.19.0
             The input to the MemoryReader now must be a np.ndarray
             Added optional velocities and forces
+        .. versionchanged:: 2.2.0
+            Input kwargs are now stored under the `_kwargs` attribute, and
```

Comment:
```suggestion
            Input kwargs are now stored under the :attr:`_kwargs` attribute, and
```
or use double backticks.